tests/scripts/thread-cert/border_router/test_firewall.py

```diff
@@ -201,6 +201,12 @@ def host_ping_ether(dest, interface, ttl=10, add_interface=False, add_route=Fals
             br1.ping_ether(router1.get_ip6_address(config.ADDRESS_TYPE.OMR)[0],
                            interface=br1.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
 
+        # 16. Host sends a UDP packet to router1's OMR address's TMF port (61631).
+        host.udp_send_host(router1.get_ip6_address(config.ADDRESS_TYPE.OMR)[0], 61631, "HELLO")
+
+        # 17. BR1 sends a UDP packet to router1's ML-EID's TMF port (61631).
```

Comment:
What about checking that BR1 can't send TMF packet to itself neither with `udp_send_host`?

Reply:
I added a new test step for this scenario. However, such packets will go to the `lo` interface which isn't captured in our testing framework. I tried capturing `lo` locally and it turned out to result in 10x packets. I'll leave this test step as is and try to implement the packet verification in a future PR.